README.md

```diff
@@ -1,119 +1,189 @@
-| InceptionResnet-V2   | 64.67 | 81.42 | 0.542 | 74.01 | 87.49 | 0.651
-| Inception-V4         | 67.45 | 82.78 | 0.560 | 73.00 | 86.87 | 0.637
-| SE-ResNeXt-101-32x4d | 72.23 | 87.28 | 0.620 | 77.13 | 89.48 | 0.693 
-| ---------------- | ---- | ---- | ---- | ---- | ---- | ---- |
-| Dataset | DF20M | DF20M | DF20M | DF20 | DF20 | DF20 | 
-
-### ViT x CNN Performance Evaluation
-Classification results of selected CNN and ViT architectures on DF20 and DF20\,-\,Mini dataset for two input resolutions [224𐄂224, 384𐄂384].
-
-|  | Top1 [%] | Top3 [%] | F1 | Top1 [%] | Top3 [%] | F1 |
-| ---------------- | ---- | ---- | ---- | ---- | ---- | ---- |
-| EfficientNet-B0     | 65.66 | 83.35 | 0.531 | 70.33 | 85.19 | 0.613
-| EfficientNet-B3     | 67.39 | 83.74 | 0.550 | 72.51 | 86.77 | 0.634
-| SE-ResNeXt-101      | 68.87 | 85.14 | 0.585 | 74.26 | 87.78 | 0.660
-| ViT-Base/16         | 70.11 | 86.81 | 0.600 | 73.51 | 87.55 | 0.655
-| ViT-Large/16        | 71.04 | 86.15 | 0.603 | 75.29 | 88.34 | 0.675
-| ---------------- | ---- | ---- | ---- | ---- | ---- | ---- |
-| Dataset | DF20M | DF20M | DF20M | DF20 | DF20 | DF20 | 
-
-|  | Top1 [%] | Top3 [%] | F1 | Top1 [%] | Top3 [%] | F1 |
-| ---------------- | ---- | ---- | ---- | ---- | ---- | ---- |
-| EfficientNet-B0  | 69.62 | 85.96 | 0.582 | 75.35 | 88.67 | 0.670
-| EfficientNet-B3  | 71.59 | 87.39 | 0.613 | 77.59 | 90.07 | 0.699
-| SE-ResNeXt-101   | 74.23 | 88.27 | 0.651 | 78.72 | 90.54 | 0.708
-| ViT-Base/16      | 74.23 | 89.12 | 0.639 | 79.48 | 90.95 | 0.727
-| ViT-Large/16     | 75.85 | 89.95 | 0.669 | 80.45 | 91.68 | 0.743
-| ---------------- | ---- | ---- | ---- | ---- | ---- | ---- |
-| Dataset | DF20M | DF20M | DF20M | DF20 | DF20 | DF20 | 
-
-### Metadata Usage Experiment
-Performance gains from Fungus observation metadata: H - Habitat, S - Substrate, M - Month, and their combinations, on DF20. 
-
-#### DF20 - ViT-Large/16 with image size 384𐄂384. 
-| H | M | S | Top1 [%] | Top3 [%] | F1 |
-| ---- | ---- | ---- | ---- | ---- | ---- |
-| 𐄂  | 𐄂  | 𐄂  |  80.45 | 91.68 | 0.743 |
-| ✔ | 𐄂  | 𐄂  | +1.50 | +1.00 | +0.027  | 
-| 𐄂  | ✔ | 𐄂  | +0.95 | +0.62 | +0.014 |
-| 𐄂  | 𐄂  | ✔ | +1.13 | +0.69 | +0.020 |
-| 𐄂  | ✔ | ✔ | +1.93 | +1.27 | +0.032 |
-| ✔ | 𐄂  | ✔ | +2.48 | +1.66 | +0.044 |
-| ✔ | ✔ | 𐄂  | +2.31 | +1.48 | +0.040 |
-| ✔ | ✔ | ✔ | +2.95 | +1.92 | +0.053 |
- #### DF20-Mini - ViT-Base/16 with image size 224𐄂224. 
-| H | M | S | Top1 | Top3 | F1 |
-| ---- | ---- | ---- | ---- | ---- | ---- |
-| 𐄂  | 𐄂  | 𐄂  | 73.51 | 87.55 | 0.655 |
-| ✔ | 𐄂  | 𐄂  | +1.94 | +1.50 | +0.040  |
-| 𐄂  | ✔ | 𐄂  | +1.23 | +0.95 | +0.020   |
-| 𐄂  | 𐄂  | ✔ | +1.39 | +1.17 | +0.025  |
-| 𐄂  | ✔ | ✔ | +2.47 | +1.98 | +0.042   |
-| ✔ | 𐄂  | ✔ | +3.23 | +2.47 | +0.062   |
-| ✔ | ✔ | 𐄂  | +3.11 | +2.30 | +0.057  | 
-| ✔ | ✔ | ✔ | +3.81 | +2.84 | +0.070 |
+## Exploiting metadata with late-fusion
+To allow further processing model predictions with available metadata we provide a notebook  `./inference/metadata_fusion.ipynb`
```

Comment:
maybe 'updating model predictions by incorporating metadata' or sth like that? Processing model predictions is not clear to me